deepchecks/core/suite.py

```diff
@@ -141,7 +141,7 @@ def _ipython_display_(
                 ),
                 raw=True
             )
-        elif is_widgets_enabled() and as_widget and not is_kaggle_env():
+        elif not is_kaggle_env() and is_interactive_output_use_possible() and as_widget:
```

Comment:
reminder why we do `not is_kaggle_env()`?

Reply:
plotly widgets aren't working in kaggle even tho it seems like kaggle has widgets, maybe with this update we don't need it?
@yromanyshyn can you check if it's still needed?

Reply:
I do not remember exactly/
whether they needed `jupyterlab-plotly` extension and it was not installed
or whether there was an incompatible version of it installed

yup I will check

Reply:
![image](https://user-images.githubusercontent.com/71635444/172593999-431d41b9-9177-489b-b096-a20ce6c842a5.png)
![image](https://user-images.githubusercontent.com/71635444/172594233-fd0716af-f174-4df4-9f12-47c334e7ff0b.png)

`go.FigureWidget` does not work on kaggle
this happens only for ploty>5.5.0